docs/source/monitoring.rst

```diff
@@ -0,0 +1,153 @@
+
+Look for ``failed: false`` in the result above. This process will exercise the core parts of |st2|. If it succeeds, then the system
+is correctly configured and working.
+
+Processes
+^^^^^^^^^
+
+You can use ``sudo st2ctl status`` to get a quick overview of current process status:
+
+.. code-block:: bash
+
+    $ sudo st2ctl status
+    ##### st2 components status #####
+    st2actionrunner PID: 1014
+    st2api PID: 921
+    st2api PID: 1285
+    st2stream PID: 922
+    st2stream PID: 1287
+    st2auth PID: 912
+    st2auth PID: 1286
+    st2garbagecollector PID: 910
+    st2notifier PID: 916
+    st2resultstracker PID: 913
+    st2rulesengine PID: 920
+    st2sensorcontainer PID: 907
+    st2chatops is not running.
+    mistral-server PID: 1031
+    mistral-api PID: 1006
+    mistral-api PID: 1228
+    mistral-api PID: 1231
+    $
+
+In a distributed system, only some of these processes will be running on each system. In the example here ``st2chatops`` is not configured on this system.
+
+Tools such as nagios or check_mk can be used to monitor the process list. Some services spawn more than one process. The exact number will depend upon your system configuration - e.g. ``st2actionrunner`` will spawn additional processes on a multi-core system. 
+
+Additional processes to monitor:
```

Comment:
possibly out of scope here, but I added "logstash" (or rsyslog, or whatever log ingest tool) to our internal processes to monitor as well